api/swagger-spec/v1.json

```diff
@@ -17624,6 +17628,40 @@
      }
     }
    },
+   "v1.AzureDiskVolumeSource": {
+    "id": "v1.AzureDiskVolumeSource",
+    "description": "AzureDisk represents an Azure Data Disk mount on the host and bind mount to the pod.",
+    "required": [
+     "diskName",
```

Comment:
Why is `diskName` required? Is this an Azure API requirement? I noticed that if the `diskName` doesn't match, I get an error... but I'm not sure why. The URL should identify a single "disk", the name should be redundant, no?